Sources/TuistAcceptanceTesting/TuistAcceptanceTestCase.swift

```diff
@@ -32,11 +32,10 @@ open class TuistAcceptanceTestCase: XCTestCase {
         derivedDataPath = try! TemporaryDirectory(removeTreeOnDeinit: true).path
         fixtureTemporaryDirectory = try! TemporaryDirectory(removeTreeOnDeinit: true)
 
-        sourceRootPath = try! AbsolutePath(
-            validating: ProcessInfo.processInfo.environment[
-                "TUIST_CONFIG_SRCROOT"
-            ]!
-        )
+        sourceRootPath = try! AbsolutePath(validating: #file.replacingOccurrences(of: "file://", with: ""))
+            .removingLastComponent()
+            .removingLastComponent()
+            .removingLastComponent()
```

Comment:
This won't work when `TuistAcceptanceTesting` is cached. And it's the reason why the env variable was introduced in the first place.

Imho, a better fix is to allow specifying an environment variable for the autogenerated schemes.